client/productOrder.jsx

```diff
@@ -5,51 +5,127 @@ class ProductOrder extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      selectedColorway: 'Thunder (Dark Blue Upper / White Sole'
+      selectedColorway: 'Thunder (Dark Blue Upper / White Sole',
+      stock8: this.props.parent_state.thunderx8Inventory,
+      stock85: this.props.parent_state.thunderx85Inventory,
+      stock9: this.props.parent_state.thunderx9Inventory,
+      stock95: this.props.parent_state.thunderx95Inventory,
+      stock10: this.props.parent_state.thunderx10Inventory,
+      stock105: this.props.parent_state.thunderx105Inventory,
+      stock11: this.props.parent_state.thunderx11Inventory,
+      stock115: this.props.parent_state.thunderx115Inventory,
+      stock12: this.props.parent_state.thunderx12Inventory,
+      stock125: this.props.parent_state.thunderx125Inventory,
+      stock13: this.props.parent_state.thunderx13Inventory,
+      stock135: this.props.parent_state.thunderx135Inventory,
+      stock14: this.props.parent_state.thunderx14Inventory
     }
+
     this.handleSubmit = this.handleSubmit.bind(this);
   }
   handleSubmit() {
 
   }
+  componentDidMount() {
+
+
+  }
+
+
+
   render() {
+    function appendSVGButton(state, shoeSize) {
+      if (state) {
```

Comment:
I notice that there is a lot of similar code between the if and else—could an inline if/else or ternary save you some repetition? Only a cosmetic note...